demisto_sdk/__main__.py

```diff
@@ -911,40 +877,61 @@ def run_playbook(**kwargs):
+    required=False)
 @click.option(
-    "-i", "--input",
+    "-j", "--json",
     help="Valid JSON file path. If not specified, the script will wait for user input in the terminal. "
          "The response can be obtained by running the command with `raw-response=true` argument.",
     required=False)
 @click.option(
-    "-p", "--prefix", help="Output prefix like Jira.Ticket, VirusTotal.IP, the base path for the outputs that the "
-                           "script generates", required=True)
+    "-p", "--prefix",
+    help="Output prefix like Jira.Ticket, VirusTotal.IP, the base path for the outputs that the "
+         "script generates", required=False)
 @click.option(
-    "-o", "--output", help="Output file path, if not specified then will print to stdout", required=False)
+    "-o", "--output",
+    help="Output file path, if not specified then will print to stdout",
+    required=False)
 @click.option(
-    "-v", "--verbose", is_flag=True, help="Verbose output - mainly for debugging purposes")
+    "-v", "--verbose", is_flag=True,
+    help="Verbose output - mainly for debugging purposes")
 @click.option(
-    "--interactive", help="If passed, then for each output field will ask user interactively to enter the "
-                          "description. By default is interactive mode is disabled", is_flag=True)
+    "--interactive",
+    help="If passed, then for each output field will ask user interactively to enter the "
+         "description. By default is interactive mode is disabled",
+    is_flag=True)
 @click.option(
     "-d", "--descriptions",
     help="A JSON or a path to a JSON file, mapping field names to their descriptions. "
          "If not specified, the script prompt the user to input the JSON content.",
     is_flag=True)
-def json_to_outputs_command(**kwargs):
+@click.option(
+    "-i", "--input",
+    help="Valid YAML integration file path.",
+    required=False)
+@click.option(
+    "-e", "--examples",
+    help="Integrations: path for file containing command examples."
+         " Each command should be in a separate line."
+         " Scripts: the script example surrounded by quotes."
+         " For example: -e '!ConvertFile entry_id=<entry_id>'")
+@click.option(
+    "--insecure",
+    help="Skip certificate validation to run the commands in order to generate the docs.",
+    is_flag=True)
+def generate_outputs(**kwargs):
     """Demisto integrations/scripts have a YAML file that defines them.
     Creating the YAML file is a tedious and error-prone task of manually copying outputs from the API result to the
     file/UI/PyCharm. This script auto generates the YAML for a command from the JSON result of the relevant API call
+    In addition you can supply examples files and generate the context description directly in the YML from those examples.
     """
-    check_configuration_file('json-to-outputs', kwargs)
-    json_to_outputs(**kwargs)
+    return run_generate_outputs(**kwargs)
```

Comment:
Please call to check_configuration_file here. Because all the calls to check_configuration_file are from main i think its better to follow this standard

Reply:
Better?